drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -283,12 +283,21 @@ static struct nrf_usbd_ctx usbd_ctx = {
 	.ready = false,
 };
 
-
 static inline struct nrf_usbd_ctx *get_usbd_ctx(void)
 {
 	return &usbd_ctx;
 }
 
+static inline bool dev_attached(void)
```

Comment:
do you have all functions inline?

Reply:
What do you mean?

Reply:
I mean we should not have all functions inlined.